graph/pkg/service/v0/drives.go

```diff
@@ -31,6 +32,14 @@ import (
 
 // GetDrives implements the Service interface.
 func (g Graph) GetDrives(w http.ResponseWriter, r *http.Request) {
+	sanitized := strings.TrimPrefix(r.URL.Path, "/graph/v1.0/")
+	// Parse the request with odata parser
+	odataReq, err := godata.ParseRequest(r.Context(), sanitized, r.URL.Query())
+	if err != nil {
+		g.logger.Err(err).Msg("unable to parse odata request")
```

Comment:
it would/will help later debugging if you added the URL.Query to the logger message.